app/packages/rust_sdk/lib/acter_flutter_sdk.dart

```diff
@@ -68,17 +68,11 @@ String userAgent = '$appName/$versionName';
 RegExp logFileRegExp = RegExp('app_.*log');
 RegExp screenshotFileRegExp = RegExp('screenshot_.*png');
 
-Color convertColor(ffi.EfkColor? primary, Color fallback) {
+Color convertColor(int? primary, Color fallback) {
```

Comment:
Arguably this is a one-liner now we might want to drop ... but that can happen at a later time ...

Reply:
@gnunicorn 
You may better know about this rust side change.

Reply:
this is the dart-side ;) ...

```
convertColor = (int? primary, color Fallback) => primary != null ? Color(primary) : fallback
```

or maybe even `Color(primary ?? fallback)` is enough?

Reply:
Yes, I think this should be good. Thanks for the suggestion :)
`Color convertColor(int? primary, Color fallback) => Color(primary ?? fallback.value);`